db/migrate/20250930120000_add_isbn_to_links.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddIsbnToLinks < ActiveRecord::Migration[7.1]
+  def change
+    add_column :links, :isbn, :string, limit: 20
```

Comment:
Some products have multiple books. You won't be able to validate the ISBN of each book with this architecture. This should have been done in the `product_files` table